build/scripts/locate-vs.ps1

```diff
@@ -13,7 +13,7 @@ function Create-Directory([string[]] $path) {
 
 function Locate-LocateVsApi {
   $packagesPath = Locate-PackagesPath
-  $locateVsApi = Join-Path -path $packagesPath -ChildPath "RoslynTools.Microsoft.LocateVS\$locateVsApiVersion\lib\net46\LocateVS.dll"
```

Comment:
I assume there is no such copy, or is that a copy if you want to link it in to another app? (i.e. you can use it as library or standalone tool)